source/targets/data_preparation/_targets.R

```diff
@@ -0,0 +1,212 @@
+################################################################################
+# targets pipeline for reading, cleaning and preparation of the MBAG MAS data  #
+# see https://books.ropensci.org/targets/                                      #
+################################################################################
+
+# Load packages required to define the pipeline
+library(targets)
+library(tarchetypes)
+library(dplyr)
+library(readr)
+library(sf)
+
+# Set target options
+tar_option_set(
+  packages = c("tidyverse",
+               "sf"),
+  format = "qs",
+  memory = "transient",
+  garbage_collection = TRUE,
+  workspace_on_error = TRUE
+)
+
+# Conflicts
+conflicted::conflicts_prefer(dplyr::filter)
+conflicted::conflicts_prefer(dplyr::select)
+conflicted::conflicts_prefer(dplyr::lag)
+
+# Set directory locations
+target_dir <- rprojroot::find_root_file(
+  "source", "targets", "data_preparation",
+  criterion = rprojroot::is_git_root)
+mbag_dir <- rprojroot::find_root_file(
+  criterion = rprojroot::is_git_root)
+
+# Source custom functions
+lapply(list.files(file.path(target_dir, "R"), full.names = TRUE), source)
+source(file.path(mbag_dir, "source", "R", "predatoren_f.R"))
+source(file.path(mbag_dir, "source", "R", "taxon_mapping.R"))
+
+# Target list
+list(
+  # 1. Read in observation data
+  tarchetypes::tar_files_input(
+    name = mas_counts_sovon_files,
+    files = paths_to_counts_sovon(
+      proj_path = target_dir
+    )
```

Comment:
This searches for "qgis_export_sovon_wfs" in "mbag-mas/source/targets/data_preparation/data". I'm not sure why you chose to place the data in that folder instead of "mbag-mas/data". Is there a specific reason?

Reply:
I will add more information in the files

Reply:
done in: [fccbf71](https://github.com/inbo/mbag-mas/pull/37/commits/fccbf7166a24fc922a57d8f3ccbde287e73bcf8e)